### PR TITLE
React-Rails Webpack

### PR DESCRIPTION
**Pull request by Timbar09**
In this PR, I created a `React-rails` app.

**Change**

- Created a new Rails app called `hello-react-rails` with a Postgres database set up. 
- Created an API endpoint that selects a random greeting from the `greetings` table.
- Created a static view that is the root of your app.
- Created the `Greeting` component that displays a greeting.
- Created a store, an action, and a reducer that connects to the API endpoint to get the random greeting.
- Display ed the random greeting in the `Greeting` component.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+| 👤 Author    | GitHub                                   | Twitter                                 | LinkedIn                                                      |
+| :----------- | :--------------------------------------- | :-------------------------------------- | :------------------------------------------------------------ |
+| Miles Mosweu | [@Timbar09](https://github.com/Timbar09) | [@Milez09](https://twitter.com/Milez09) | [@miles-mosweu09](https://www.linkedin.com/in/miles-mosweu09) |
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Add more features.**
+
```
Comment (HAMMAS-SALEEM) [inline README.md]: - [ ] kindly put some meaningful features in the Future Feature section to make your readme descriptive and professional 👍

**Review by HAMMAS-SALEEM — COMMENTED** [review]
# Invalid Submission :red_circle:

invalid-submission

## Hey @Timbar09 :wave:

Good job on implementing some of the requirements but I cannot proceed review of your pull request because there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval. Still, we highly recommend to students that they should take these changes into consideration._

Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by iambenkis — APPROVED** [review]
Hi @Timbar09 🤝 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
👍🏽 Addressed all requested changes ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Timbar09** [comment]
Thanks for the feedback, @iambenkis, and @HAMMAS-SALEEM.

**Pull request merged**